deps/winpty/src/agent/Agent.cc

```diff
@@ -223,10 +223,18 @@ Agent::Agent(LPCWSTR controlPipeName,
 
 Agent::~Agent()
 {
-    trace("Agent::~Agent entered");
-    agentShutdown();
-    if (m_childProcess != NULL) {
-        CloseHandle(m_childProcess);
+    trace("Agent::~Aent entered");
```

Comment:
```suggestion
    trace("Agent::~Agent entered");
```

Looks like a “g“ was accidentally lost here

Reply:
Thanks! https://github.com/microsoft/node-pty/pull/722